pkg/datapath/linux/requirements.go

```diff
@@ -219,7 +219,12 @@ func CheckMinRequirements() {
 	if !option.Config.DryMode {
 		probeManager := probes.NewProbeManager()
 		if err := probeManager.SystemConfigProbes(); err != nil {
-			log.WithError(err).Warning("BPF system config check: NOT OK.")
+			switch err {
```

Comment:
I think it'd be more safe and idiomatic to define an error type, and then do a type assertion for that type here.